spring-cloud-dataflow-admin/src/main/java/org/springframework/cloud/dataflow/admin/controller/StreamController.java

```diff
@@ -80,7 +81,7 @@
 	/**
 	 * The module registry this controller will use to look up modules.
```

Comment:
this javadoc needs to be updated: "module" -> "artifact" and "look up modules" -> "look up modules and libraries"